packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/FormulaDetails.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { theme } from "@cdt/ui";
+import styled from "styled-components";
+import MathJax from "react-mathjax-preview";
+
+import { ErrorBoundary } from "../../../common/ErrorBoundary";
+import { Summary } from "../../common/stepStyles";
+
+function FormulaDetails({ values, formula }) {
+  return (
+    <Details>
+      <Summary>Voir le detail du calcul</Summary>
+      <HeadingDetails>Élements saisis :</HeadingDetails>
+      <ListDetails>
+        {Object.entries(values).map(([label, value], index) => (
+          <ItemDetails key={index}>
+            {label}&nbsp;: {value}
+          </ItemDetails>
+        ))}
+      </ListDetails>
+      <ErrorBoundary>
+        <HeadingDetails>Formule</HeadingDetails>
+        <FormuleWrapper>
+          <MathJax math={"`" + formula + "`"} />
+        </FormuleWrapper>
+      </ErrorBoundary>
+    </Details>
+  );
+}
+
+export { FormulaDetails };
+
+FormulaDetails.propTypes = {
+  values: PropTypes.object.isRequired,
+  formula: PropTypes.string.isRequired
+};
+
+const { spacing, fonts } = theme;
+
+const Details = styled.details`
+  margin-top: ${spacing.interComponent};
+`;
+
+const HeadingDetails = styled.strong`
+  font-weight: bold;
+  font-size: ${fonts.sizeSmall};
+`;
+
+const ListDetails = styled.ul`
+  list-style-image: url("data:image/svg+xml;,${encodeURIComponent(
```

Comment:
Do you really need an SVG here ?

Reply:
ahah, the design says it's a "-" so I put a "-" (I LOOOOOOVE SVG)